src/reduxTypes.js

```diff
@@ -256,6 +256,7 @@ export type RealmState = {|
   twentyFourHourTime: boolean,
   canCreateStreams: boolean,
   isAdmin: boolean,
+  isTopicMandatory: boolean,
```

Comment:
See the note in the jsdoc on this `RealmState` type about how the properties are organized: stuff about the server, then about the org/realm, then about the user.

This property is about the org aka the realm -- that's why its name in the API is prefixed `realm_` and it's controlled by the `realm` event type -- so it belongs in the "realm" section.

(I realize this is kind of easy to miss. I may try moving that jsdoc material inline into the type definition, next to the respective properties.)